### PR TITLE
added athena median support

### PR DESCRIPTION
**Pull request by kuhnen**
Changes proposed in this pull request:
- The proposal of this pull request is to use the approx_percentile function to calculate the median when using athena as the backend

image

https://prestodb.io/docs/current/functions/aggregate.html

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open-source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in GitHub issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

Thank you for submitting!

### CODE REVIEW

**Comment by eugmandel** [comment]
@kuhnen Thank you for creating this PR! We will review it shortly.

File: great_expectations/dataset/sqlalchemy_dataset.py [hunk 1]
```diff
@@ -784,36 +784,42 @@ def get_column_unique_count(self, column):
...
-            sa.select([sa.column(column)])
-            .order_by(sa.column(column))
-            .where(sa.column(column) != None)
-            .offset(max(nonnull_count // 2 - 1, 0))
-            .limit(2)
-            .select_from(self._table)
-        )
+            element_values = self.engine.execute(
+                f"SELECT approx_percentile({column},  0.5) FROM {self._table}"
+            )
```
Comment (Shinnnyshinshin) [inline great_expectations/dataset/sqlalchemy_dataset.py:791]: Thank you very much for this enhancement @kuhnen, LGTM

**Review by Shinnnyshinshin — APPROVED** [review]

File: docs/changelog.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Changelog
 
 Develop
 -----------------
-
+* [ENHANCEMENT] Improve support for median calculation in Athena
```
Comment (Shinnnyshinshin) [inline docs/changelog.rst]:
```suggestion
* [ENHANCEMENT] Improve support for median calculation in Athena (Thanks @kuhnen!) #2521

```

Comment (Shinnnyshinshin) [reply docs/changelog.rst]: and requesting one small addition to the changelog entry

**Comment by kuhnen** [comment]
Thanks, @Shinnnyshinshin.  I am very excited to have my first pull request approved. I just rebased with upstream/develop.  Please let me know if  I need to do any extra changes.
Let's rock with Great Expectations.

**Pull request merged**